packages/tools/fetch-tool/src/fluidFetchMessages.ts

```diff
@@ -140,16 +185,19 @@ async function* saveOps(
     // Split into 100K ops
     const chunk = 100 * 1000;
 
+    let sequencedMessages: ISequencedDocumentMessage[] = [];
+
     // Figure out first file we want to write to
     let index = 0;
     if (files.length !== 0) {
-        index = (files.length - 1);
+        index = files.length - 1;
     }
+    // current sequence number to start writing from
     let curr = index * chunk + 1;
```

Comment:
Why did you revert your change to initialize curr to first op seq number of last file? I don't think this is right? Can you please tell me the reason for the reverting, maybe I am missing something?

Reply:
As we are starting to write the last file, the first interesting op should be the first op of that file. Previously all ops were there then it was right but now it does not need to be index * chunk + 1 due to ops deletion.

Reply:
This is for the case files are present, otherwise it should just be firstAvailableDelta.

Reply:
i reverted because you said currSeq is not needed